Src/Penneo/PenneoSetup.cs

```diff
@@ -112,16 +112,16 @@ public virtual void InitializeMappings()
                   .Map(x => x.Role)
                   .Map(x => x.Conditions)
                   .Map(x => x.SignOrder)
-                  .Map(x => x.ActiveAt)
-                  .Map(x => x.ActivatedAt)
-                  .Map(x => x.ExpireAt)
+                  .Map(x => x.ActiveAt, convert: x => TimeUtil.ToUnixTime((DateTime) x))
+                  .Map(x => x.ActivatedAt, convert: x => TimeUtil.ToUnixTime((DateTime) x))
+                  .Map(x => x.ExpireAt,convert: x => TimeUtil.ToUnixTime((DateTime) x))
                   .ForUpdate()
                   .Map(x => x.Role)
                   .Map(x => x.Conditions)
                   .Map(x => x.SignOrder)
-                  .Map(x => x.ActiveAt)
-                  .Map(x => x.ActivatedAt)
-                  .Map(x => x.ExpireAt)
+                  .Map(x => x.ActiveAt, convert: x => TimeUtil.ToUnixTime((DateTime) x))
+                  .Map(x => x.ActivatedAt, convert: x => TimeUtil.ToUnixTime((DateTime) x))
+                  .Map(x => x.ExpireAt, convert: x => TimeUtil.ToUnixTime((DateTime) x))
```

Comment:
Without being an expert in C# woulden't it make sense to move the type declaration to the parameter declaration like this:

```suggestion
                  .Map(x => x.ExpireAt, convert: DateTime x => TimeUtil.ToUnixTime(x))
```

Or maybe:

```suggestion
                  .Map(x => x.ExpireAt, convert: (DateTime x) => TimeUtil.ToUnixTime(x))
```

I don't know if it has any performance impact but at least the compile would stand a chance to optimize for it 😅

Reply:
Doing this will give you an error of `Incompatible anonymous Function Signature`
I haven't looked in to it too much but I can't figure out how we could type hint as you suggest without refactoring the MappingBuilder ¯\_(ツ)_/¯

Reply:
Fair enoght. Its most likely due to the signature of `Map()`. Let it stay - I was just wondering